docs/java-rest/high-level/watcher/start-watch-service.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[java-rest-high-watcher-start-watch-service]]
```

Comment:
These have a new tidier format. See #34203 for a *much* more concise version of the docs. Pls do this for both of the asciidoc pages you have added

Reply:
Done.